src/DependencyInjection/SetonoSyliusOrderEditExtension.php

```diff
@@ -12,6 +13,8 @@
 
 final class SetonoSyliusOrderEditExtension extends Extension implements PrependExtensionInterface
 {
+    use PrependDoctrineMigrationsTrait;
```

Comment:
I never use these in my own projects. I find them invasive somehow :D Also I don't understand how they can all be compliant with all database engines.

I would rather not have them, but if you think it's a good idea, I am fine with that.

Reply:
Usage of migrations configured in such a way makes it easier to use the plugin (one step less, the end-user does not have to generate migrations by themselves) and from my feeling makes a plugin more reliable. But, it's probably my bubble of habit :dancer: If you don't have migrations generated in other plugins and would like this to be consistent with them - let be it this way 🫡